### PR TITLE
adds a new hostile simplemob the yeti

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
wanted to get better at doing icons and decided to use some sprites from goon 2016 and make a sprite 
as i am unable to find the creator heres the link to it https://github.com/goonstation/goonstation-2016/blob/d8a2d60915fd3b74653a1b7d4b8a0910c6fc2f19/icons/misc/critter.dmi
https://cdn.discordapp.com/attachments/134720091576205312/674841407726551040/unknown.png
you can encounter this ruin in the lavaland winter dome and through xenobio gold slimes
#### Changelog

:cl:  
rscadd: adds a new hostile simplemob you can encounter through xenobio or through ruins  
/:cl:

### CODE REVIEW

File: yogstation.dme [hunk 1]
```diff
@@ -2221,6 +2221,7 @@
 #include "code\modules\mob\living\simple_animal\hostile\wizard.dm"
 #include "code\modules\mob\living\simple_animal\hostile\wumborian_fugu.dm"
 #include "code\modules\mob\living\simple_animal\hostile\zombie.dm"
+#include "code\modules\mob\living\simple_animal\hostile\yeti.dm"
```
Comment (monster860) [inline yogstation.dme:2223]: y should got before z I'm pretty sure

**Review by monster860 — CHANGES_REQUESTED** [review]

File: yogstation.dme (hunk 1 above)
Comment (boodaliboo) [reply yogstation.dme:2223]: whoops

File: code/modules/mob/living/simple_animal/hostile/yeti.dm [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	speed = 3
+	butcher_results = list(/obj/item/stack/sheet/bone = 5)
+	response_harm   = "smacks"
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	attacktext = "slams"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	faction = list("skeleton")
+
+	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
```
Comment (monster860) [inline code/modules/mob/living/simple_animal/hostile/yeti.dm]: Why shouldn't yetis take CO2 damage? (the default is this but with co2 damage)

**Review by monster860 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/yeti.dm (hunk 2 above)
Comment (boodaliboo) [reply code/modules/mob/living/simple_animal/hostile/yeti.dm]: they spawn in a lavaland ruin in which i cant remeber if it was pressurized at all

File: code/modules/mob/living/simple_animal/hostile/yeti.dm [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	speed = 3
+	butcher_results = list(/obj/item/stack/sheet/bone = 5)
+	response_harm   = "smacks"
+	melee_damage_lower = 10
+	melee_damage_upper = 15
+	attacktext = "slams"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	faction = list("skeleton")
+
+	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
```
Comment (monster860) [inline code/modules/mob/living/simple_animal/hostile/yeti.dm]: nice now since this is exactly identical to the default value just remove this line.

**Review by monster860 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/yeti.dm (hunk 3 above)
Comment (boodaliboo) [reply code/modules/mob/living/simple_animal/hostile/yeti.dm]: i messed up

**Comment by alexkar598** [comment]
Pointless

**Comment by overlord12345** [comment]
Would be better if you change the attack text into '' slashes'' or "claws at" as the yeti has claws

**Comment by overlord12345** [comment]
Also cause he has stick arms maybe you can call him a '' angry stick yeti"

**Comment by boodaliboo** [comment]
> Pointless

and? many things are pointless give actual criticism

**Comment by boodaliboo** [comment]
> Also cause he has stick arms maybe you can call him a '' angry stick yeti"

it will be done

**Comment by overlord12345** [comment]
Nice

**Comment by overlord12345** [comment]
Also at the moment it dose less damage then a undead eskimo, is it meant to be weaker or stronger then them?

**Comment by overlord12345** [comment]
> Pointless

its not pointless it adds something to gaurd the snow dome other them the two skeletons, and makes getting a cap laser gun and chainsaw a bit harder then before

**Comment by boodaliboo** [comment]
I left it low and I can always buff it later

**Review by nesisknesis — COMMENTED** [review]
Would prefer if this was a chance spawn rather than a guaranteed spawn. It's a nice reference but because of the nature of the sprite I wouldn't want to see it constantly. 

but thats just me

Also, rename it to Abominable Snow Monster to keep with the reference.

**Comment by boodaliboo** [comment]
Yeah sure haven’t done probability spawns before so it may take a bit and rename will be done

**Comment by nesisknesis** [comment]
Lavaland mobs like the magamwing use probability to spawn IIRC

**Comment by boodaliboo** [comment]
Thanks I’ll look into it

**Comment by alexkar598** [comment]
Since it's used on ruin, my only objection is the out of place sprite now

**Comment by boodaliboo** [comment]
The mob will be a chance thing so it’s more of a surprise when you do see it and so normally you won’t see this out of place mob in a ruin I’m thinking like a 10% chance

**Comment by boodaliboo** [comment]
currently can't figure out how to make it a effect to spawn/not spawn without affecting all snow tiles because the only example of rng spawns i can find is lavaland mobs spawning on lavaland tiles

**Review by nesisknesis — COMMENTED** [review]
The spawner for the kitchen animals uses probability, steal code from that

**Comment by boodaliboo** [comment]
Your the best

**Comment by boodaliboo** [comment]
its a probability now

**Comment by boodaliboo** [comment]
Now I just need to put on a map

File: code/modules/mob/living/simple_animal/hostile/yeti.dm [hunk 4]
```diff
@@ -0,0 +1,35 @@
+/mob/living/simple_animal/hostile/yeti
+	name = "Angry Stick Yeti"
+	desc = "It's a horrifyingly enormous yeti, and it seems to resemble something nostalgic"
```
Comment (Ktlwjec1) [inline code/modules/mob/living/simple_animal/hostile/yeti.dm]: Full stop ->something nostalgic."

**Comment by Ktlwjec1** [comment]
goon/icons/obj/yeti.dmi

Should it be "goon/icons/mob/yeti.dmi"? Makes more sense to me.

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]
- goon/icons/obj/yeti.dmi -> Should it be "goon/icons/mob/yeti.dmi"? Makes more sense to me.
- Intentional to use the mob itself, not the spawner? Spawner isn't placed on the map currently.
- You could move "yeti.dm" into "stickman.dm" completely.
- Change spawner name to something like "name = "Snow mobs" since it's not just a yeti that's included.
- There is a massive space in "response_harm   = "smacks"" before the "=" which is going to upset me in the future.

**Comment by boodaliboo** [comment]
no inspiration to finsih

**Pull request closed without merging**